views/stats-proofofplay-page.twig

```diff
@@ -69,6 +69,59 @@
 
 {% block javaScript %}
     <script type="text/javascript">
+
+        function statsExportFormSelect(dialog) {
+
+            var fromDtLink;
+            var toDtLink;
+            var displayId;
+
+            $(dialog).on("change", function() {
```

Comment:
Using the "on change" event for the main dialog object can lead to some problems, maybe we need to specify the elements we want to be triggered by this listener, something like:

`$(dialog).find("input, select").on("change", function() { ...`